libs/SalesforceAnalytics/src/com/salesforce/androidsdk/analytics/security/Encryptor.java

```diff
@@ -210,6 +210,30 @@ public static String decrypt(byte[] data, String key, byte[] iv) {
         return null;
     }
 
+    /**
+     * Decrypts data with key using using AES/GCM/NoPadding. The data is not Base64 encoded.
+     *
+     * @param data Data.
+     * @param key Key.
+     * @return Decrypted data.
+     */
+    public static byte[] decryptWithoutBase64Encoding(byte[] data, String key) {
```

Comment:
Added new methods for encryption and decryption that takes `byte[]` and returns `byte[]` without `Base64` encoding and decoding.